plugins/scaffolder-backend/src/scaffolder/tasks/StorageTaskBroker.ts

```diff
@@ -202,6 +205,32 @@ export class StorageTaskBroker implements TaskBroker {
     });
   }
 
+  public async recoverTasks(): Promise<boolean> {
+    const enabled =
+      (this.config &&
+        this.config.getOptionalBoolean(
+          'scaffolder.EXPERIMENTAL_recoverTasks',
+        )) ??
+      false;
+
+    if (enabled) {
+      const recoveredTaskIds =
```

Comment:
Maybe let's shallow this nesting a little bit and break it out into mutliple lines for readablility.